packer/ansible/roles/remoterepos/tasks/main.yml

```diff
@@ -69,6 +69,12 @@
   args:
     chdir: "{{ ansible_env.HOME }}/src/on-http"
 
+#workaround for https://github.com/RackHD/RackHD/issues/357
+- name: install dev dependencies for on-http to allow taskdoc to work
+  shell: "npm install"
```

Comment:
@WangWinson has submitted the PR (https://github.com/RackHD/on-tasks/pull/303) to not rely on dev dependencies, so I don't think we need this.